src/test/java/com/google/cloud/spanner/hibernate/SpannerTableExporterTests.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+package com.google.cloud.spanner.hibernate;
+
+import org.hibernate.boot.Metadata;
+import org.hibernate.boot.MetadataSources;
+import org.hibernate.boot.registry.StandardServiceRegistry;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.tool.hbm2ddl.SchemaExport;
+import org.hibernate.tool.schema.TargetType;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.EnumSet;
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for SpannerTableExporter.
+ *
+ * @author Chengyuan Zhao
+ */
+public class SpannerTableExporterTests {
+
+    private Metadata metadata;
+
+    private StandardServiceRegistry registry;
+
+    @Before
+    public void setup() {
+        this.registry = new StandardServiceRegistryBuilder()
+                .applySetting("hibernate.dialect", SpannerDialect.class.getName())
+                .build();
+        this.metadata = new MetadataSources(this.registry)
+                .addAnnotatedClass(TestEntity.class)
+                .buildMetadata();
+    }
+
+    @Test
+    public void generateDropStringsTest() throws IOException {
+        String testFileName = UUID.randomUUID().toString();
+        new SchemaExport().setOutputFile(testFileName).drop(EnumSet.of(TargetType.STDOUT, TargetType.SCRIPT),
```

Comment:
Is it possible to just stream the output only to STDOUT, and then verify contents form STDOUT? If you can restrict it to STDOUT, it would save you complexity of also creating/deleting the script file.

Reply:
I don't want to deal with anything else that might be going into STDOUT though. Tests may not necessarily run in series or by themselves ?

Reply:
Ok, make sense.